packages/fxa-auth-server/scripts/cleanup-secondary-emails.mjs

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env node
```

Comment:
Any reason not to write this in typescript? We will just have to convert it later on, if you don't...